eval/final/components.py

```diff
@@ -1,15 +1,13 @@
 # type: ignore
 # pylint: disable=no-value-for-parameter,import-outside-toplevel,import-error
 
-from kfp.dsl import Artifact, Output, component
+from kfp.dsl import Artifact, Input, Metrics, Output, component
```

Comment:
See below, if you intend to remove `Output[Artifact]`, the `Artifact` and `Output` imports are no longer needed here.